packages/protocol/migrationsConfig.js

```diff
@@ -37,6 +37,12 @@ const defaultConfig = {
     adjustmentSpeed: 1 / 2,
     infrastructureFraction: 1 / 2,
   },
+  quorum: {
+    participationBaseline: 8 / 10,
```

Comment:
How were these chosen?

Reply:
I believe this is the starting value Tezos chose, but it's somewhat arbitrary, and the effect of this choice presumably goes away after a few proposals.